src/olympia/addons/models.py

```diff
@@ -1129,7 +1129,9 @@ def can_request_review(self):
         latest_version = self.find_latest_version(
             amo.RELEASE_CHANNEL_LISTED, exclude=(amo.STATUS_BETA,))
 
-        return latest_version is not None and latest_version.files.exists()
+        return (latest_version is not None and
+                not latest_version.is_webextension and
+                latest_version.files.exists())
```

Comment:
Hmm, so this function is also used for the 'my submissions' list. Meaning anyone submitting a webextension will lose the ability to resume a submission.